### ISSUE TITLE
[BUG] Decky loader starts frontend plugin two times when using option reload plugin

### ISSUE THREAD

**Issue by steam3d** [SteamDeckHomebrew/decky-loader#611]
### Please confirm

- [X] I have searched existing issues
- [X] This issue is not a duplicate of an existing one
- [X] I have checked the [common issues section in the readme file](https://github.com/SteamDeckHomebrew/decky-loader#-common-issues)
- [X] I have attached logs to this bug report (failure to include logs will mean your issue may not be responded to).

### Bug Report Description

When I use the reload option, the decky run the plugin then immediately call onDismount from fronted and then again run the fronted of the plugin.

### Expected Behaviour

The frontend of the plugin start only once.

### SteamOS version

SteamOS 3.5.17 Stable

### Selected Update Channel

Stable

### Backend Logs

[deckylog.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14884208/deckylog.txt)

### Frontend Logs

[cef_log.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14884244/cef_log.txt)
